docs/content/configuring-npm/install.md

```diff
@@ -6,17 +6,27 @@ description: Download and install node and npm
 
 ### Description
 
-To publish and install packages to and from the public npm registry, you must install Node.js and the npm command line interface using either a Node version manager or a Node installer. **We strongly recommend using a Node version manager to install Node.js and npm.** We do not recommend using a Node installer, since the Node installation process installs npm in a directory with local permissions and can cause permissions errors when you run npm packages globally.
+To publish and install packages to and from the public npm registry, you
+must install Node.js and the npm command line interface using either a Node
+version manager or a Node installer. **We strongly recommend using a Node
+version manager to install Node.js and npm.** We do not recommend using a
+Node installer, since the Node installation process installs npm in a
+directory with local permissions and can cause permissions errors when you
+run npm packages globally.
 
 ### Overview
 
-- [Checking your version of npm and Node.js](#checking-your-version-of-npm-and-node-js)
-- [Using a Node version manager to install Node.js and npm](#using-a-node-version-manager-to-install-node-js-and-npm)
-- [Using a Node installer to install Node.js and npm](#using-a-node-installer-to-install-node-js-and-npm)
+- [Checking your version of npm and
```

Comment:
the hard wraps make it very awkward to review and presumably edit the text, and also makes git diffs much noisier; is there a reason it doesn’t soft wrap?

Reply:
We do hard wraps everywhere else in the docs.  In vim, it definitely makes it easier to edit, and `gq{` will re-wrap an entire paragraph easily.  I typically view markdown diffs with the "rich diff" mode, so whitespace is ignored and you can just read the rendered text more easily.

I would only be mildly opposed to moving to soft wraps (since I'd have to retrain some vim settings and muscle memory) but that's a separate conversation, and in the meantime, consistency is the way to go, imo.